modules/dxp/apps/scim/scim-rest-impl/src/main/java/com/liferay/scim/rest/internal/util/ScimUtil.java

```diff
@@ -487,5 +486,7 @@ private static void _validate(ScimUser scimUser) throws Exception {
 
 	private static final DCLSingleton<AttributeSchema>
 		_attributeSchemaDCLSingleton = new DCLSingleton<>();
+	private static final DateFormat _dateFormat = new SimpleDateFormat(
```

Comment:
@daiazius Why we can not deal with the birthday as we deal with the others `Date` attributes?

Reply:
@rafaprax the birthday Date is parsed in a different way in the REST builder generated classes and I was getting parse exceptions because of that, so I changed it here for it to work

Reply:
As discussed  in person, we need to reflect this change on SCIM's documentation, Thank you @daiazius